routers/api/v1/admin/user.go

```diff
@@ -131,6 +138,11 @@ func EditUser(ctx *context.APIContext, form api.EditUserOption) {
 	}
 
 	if len(form.Password) > 0 {
+		if !util.CheckPasswordComplexity(form.Password) {
+			err := errors.New("PasswordComplexity")
+			ctx.Error(500, "PasswordComplexity", err)
```

Comment:
5xx  errors [mean](https://restfulapi.net/http-status-codes/) something is wrong on the server side. A 4xx error means the user has done something wrong. I'd use "400 Bad Request".